conf/config-schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "object",
+    "properties": {
+        "root": { "type": "boolean" },
+        "globals": { "type": ["object", "array"] },
+        "global": { "type": ["object", "array"] },
+        "parser": { "type": ["string", "null"] },
+        "env": { "type": "object" },
+        "plugins": { "type": ["array"] },
+        "settings": { "type": "object" },
+        "extends": { "type": ["string", "array"] },
+        "rules": { "type": "object" },
+        "parserOptions": { "type": "object" }
+    },
+    "additionalProperties": false
```

Comment:
I was saying that although ecmaFeatures currently doesn't do anything, it's possible that a lot of people have it in their configs, so it might be better to just ignore it to avoid breaking a bunch of peoples' builds. But I'm fine with not allowing it.